### DISCUSSION TITLE
Separating a series of an exam in two exams

### DISCUSSION THREAD

**Discussion by thiagobusa** [dcm4che/dcm4chee-arc-light#3550] [General]
There is the possibility of separating a series of an exam into another exam.

Example: technician made two worklist items in the same exam. I now need to separate them.

**Comment by vrindanayak** [comment]
Please elaborate your use case and workflow a bit. If the examination of a patient corresponds to one procedure, then the second worklist should ideally be canceled or discontinued, in order that the series of an examination is associated only with one worklist.
